src/main/kotlin/lt/neworld/vd2svg/converter/Converter.kt

```diff
@@ -86,17 +86,25 @@ class Converter(val colors: ResourceCollector) {
     private fun Element.fixStroke() {
         val strokeColor = attributes.get(ANDROID_NS, "strokeColor")
         val strokeWidth = attributes.get(ANDROID_NS, "strokeWidth")
+        val strokelinecap = attributes.get(ANDROID_NS, "strokeLineCap")
+        val strokelinejoin = attributes.get(ANDROID_NS, "strokeLineJoin")
 
         if (strokeColor != null) {
             setAttribute("stroke", strokeColor)
+            setAttribute("stroke-linecap", strokelinecap)
+            setAttribute("stroke-linejoin", strokelinejoin)
```

Comment:
One last thing. `attributes.get(ANDROID_NS, "strokeLineCap")` will return `null` and in many cases writing `null` in attribute leads to crashes or other problems. For this reason, I added checks for each attribute, like `if (strokeColor != null) {`. So you should follow this pattern, like:
```
if (strokelinecap != null) {
 setAttribute("stroke-linecap", strokelinecap)
}
```

Everything else seems OK and I will publish it as soon as you fix this last one problem.

Reply:
Sorry for taking your time but I don't know kotlin well, i hope i was able to solve the problems completely this time.